### PR TITLE
Add Integration tests

### PR DESCRIPTION
**Pull request by murilommen**
- added integration tests for existing helpers
- added README.md
- using whylabs_client for all requests

### CODE REVIEW

File: tests/test_smoke.py [hunk 1]
```diff
@@ -1,16 +1,2 @@
-import pytest
-    
-
-def test_error_without_env_var() -> None:
```
Comment (FelipeAdachi) [inline tests/test_smoke.py:4]: just curious: why did you remove this test?

File: tests/test_entity_schema.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+    update_discreteness.update()
+
+    assert update_discreteness.current_entity_schema["columns"]["temperature"]["discreteness"] == "continuous"
+    assert update_discreteness.current_entity_schema["columns"]["prediction_temperature"][
+               "discreteness"] == "discrete"
+
+    columns_discrete = ColumnsDiscreteness(
+        discrete=["temperature"],
+        continuous=["prediction_temperature"]
```
Comment (FelipeAdachi) [inline tests/test_entity_schema.py:68]: what happens if you set the same column as discrete and continuous?

File: README.md [hunk 3]
```diff
@@ -1 +1,112 @@
...
+```bash
+make setup && poetry shell
+```
+
+## Usage
+In here we will describe some examples on how to use the package. You can also check [the integration tests directory](./tests/) to have more insights on how things are built to be used.
+### Models
+Users can change their model type between `REGRESSION`, `CLASSIFICATION` and `EMBEDDINGS`, using the models helpers, as the example shows:
+```python
+from whylabs_toolkit.helpers.models import update_model_metadata
```
Comment (andyndang) [inline README.md:19]: Why do we make imports so nested? Can it be something like:

```
from whylabs_toolkit import datasets
```

And then you can have:

```
dataset.create(...)
```
And
```
daaset.update(...)
```

File: README.md [hunk 4]
```diff
@@ -1 +1,112 @@
...
+    org_id="org_id"
+)
+
+update_entity.update()
+
+```
+#### Data types
+```python
+columns_schema = {
+    "temperature": "fractional",
```
Comment (andyndang) [inline README.md]: how do we know what datatypes are supported? THe values should be enums here

File: README.md [hunk 5]
```diff
@@ -1 +1,112 @@
...
+    UpdateColumnClasses, 
+    UpdateEntityDataTypes, 
+    ColumnsClasses,
+    UpdateColumnsDiscreteness,
+    ColumnsDiscreteness
+)
+
+# Note that you don't need to specify all existing columns, but only those you wish to modify
+
+classes = ColumnsClasses(
```
Comment (andyndang) [inline README.md]: rename `Classes` -> `Classifiers`. Class is an overloaded term

File: README.md [hunk 6]
```diff
@@ -1 +1,112 @@
...
+    inputs=[],
```
Comment (andyndang) [inline README.md]: Do I need to specify something blank?

File: README.md [hunk 7]
```diff
@@ -1 +1,112 @@
...
+    org_id="org_id"
+)
+
+update_data_types.update()
+```
+#### Discreteness
+```python
+columns_discrete = ColumnsDiscreteness(
+    discrete=["temperature"],
+    continuous=[]
```
Comment (andyndang) [inline README.md]: Same here. Do I need to specify this?

File: README.md [hunk 8]
```diff
@@ -1 +1,112 @@
...
+)
+
+update_discreteness = UpdateColumnsDiscreteness(
+    dataset_id="dataset_id",
+    classes=columns_discrete,
```
Comment (andyndang) [inline README.md]: `classes=columns_discrete` looks rather odd

**Review by andyndang — COMMENTED** [review]
Also consider doing a builder pattern:

```
client = WhyLabsClient(....) // configure org ID + api key
datasets = DatasetHelper().with_client(client)
datasets.create(...) //  no need to respecify org ID
datasets.delete(...)

schema = datasets.get_schema_manager(dataset_id=...)
schema.update_discreteness(...)
schema.update_classifier(...)


monitors = datasets.get_monitor_manager(dataset_id=...)
monitors.list()
monitors.dumps()
monitors.create(...)
monitors.delete(..)

File: tests/test_smoke.py (hunk 1 above)
Comment (murilommen) [reply tests/test_smoke.py:4]: with the environment variable set via Git Actions, it would be cumbersome to have this variable deleted system-wide and then re-defined to be used on other tests.

File: README.md (hunk 4 above)
Comment (murilommen) [reply README.md:72]: the values are validated by whylabs_client's enums. but maybe it's best to build an enum on whylabs_toolkit too

File: README.md (hunk 6 above)
Comment (murilommen) [reply README.md]: not necessarily. will change the example to make this clear

File: README.md (hunk 3 above)
Comment (murilommen) [reply README.md:19]: yes, definitely. i will defer this to the next PR, which is dedicated to refactor the codebase pattern + making imports look nicer.

File: tests/test_entity_schema.py (hunk 2 above)
Comment (murilommen) [reply tests/test_entity_schema.py:68]: good point, will add this test

**Review by andyndang — APPROVED** [review]
Ship it
